src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+export interface TabType {
+  id: string;
+  title: string;
+  content: string;
+}
+
+type TabListProps = {
+  tabs: TabType[];
+  handleTabClick: (tabItem: TabType) => void;
+  activeTab: TabType;
+};
+
+export const Tabs: React.FC<TabListProps> = ({
+  tabs,
+  handleTabClick,
+  activeTab,
+}) => {
+  return (
+    <>
+      <ul>
+        {tabs.map((tabItem) => (
+          <li
+            key={tabItem.id}
+            className={tabItem === activeTab ? 'is-active' : ''}
+            data-cy="Tab"
+          >
+            <a
+              href={`#${tabItem.id}`}
+              data-cy="TabLink"
+              onClick={(e) => {
+                e.preventDefault();
+                handleTabClick(tabItem);
+              }}
```

Comment:
Move the onClick logic to a separate function. It follows the best practice of separating concerns and keeping your JSX code clean and focused on rendering components rather than defining complex logic directly within it.